packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js

```diff
@@ -224,6 +224,26 @@ QUnit.test('Labels offsets', function(assert) {
     assert.deepEqual(yDifference, [30, 30, 30], 'vertical offset applied');
 });
 
+QUnit.test('label horizontalOffset option should be updated and re-painted on runtime. (T1243009)', function(assert) {
+    const instance = createSankey({
+        dataSource: [{ source: 'A', target: 'Z', weight: 1 }, { source: 'B', target: 'Z', weight: 1 }],
+        label: {
+            horizontalOffset: 0,
+        }
+    });
+
+    const x1 = this.labels().map(function(label) { return label.children[0].attr.lastCall.args[0].translateX; });
+
+    instance.option('label', {
+        horizontalOffset: 30,
+    });
+
+    const x2 = this.labels().map(function(label) { return label.children[0].attr.lastCall.args[0].translateX; });
```

Comment:
Don't you want to create a getter function to reduce copypaste?

Reply:
i will try

Reply:
[5c2f6d9](https://github.com/DevExpress/DevExtreme/pull/27756/commits/5c2f6d924fc31fa76ad46902858feee53ba83ee5)